### PR TITLE
Initial list of topics that are out of scope

### LINKED ISSUE #88: What's out of scope? (was: This is not congestion control)
Make clear time scale is way too long for this to be used as congestion controll

Make clear this is not a signal of congestion

### PR DESCRIPTION
**Pull request by SpencerDawkins**
I'm SURE there will be more, but at least we have these as a starting point!

closes #88

### CODE REVIEW

**Comment by mjoras** [comment]
I think this looks good. Though we are making the charter ever-longer, I'm not sure there's any other way to address these points.

**Review by mjoras — APPROVED** [review]

**Comment by SpencerDawkins** [comment]
> I think this looks good. Though we are making the charter ever-longer, I'm not sure there's any other way to address these points.

- Based on my understanding of discussions on specific issues/PRs with @martinthomson, I THINK Martin is right about not wanting to say too much in a proposed charter sent for balloting. 
- Based on my understanding of the conversation on the July 10 call coordination call with @zaheduzzaman, I THINK his expectation is that it's OK for us to say more in a pre-BOF proposed charter than in a post-BOF proposed charter sent for balloting. 

Suggestions for @martinthomson and @fluffy - 

- When we are discussing charter text (especially on topics that come up every time we turn a corner at an IETF meeting), we consider flagging "useful background for charter discussions but delete it before approving the charter" text, the way we do with notes to the RFC Editor about removing implementation sections, etc. before RFC publication. 
- We are still getting suggestions to address topics in the charter text that we (at least, @SpencerDawkins) has marked as "InADeliverable" in GitHub issues. The goal for that label is that it acknowledges a topic that matters, but we're expecting to defer decisions about that topic until the working group is formed, because we think the working group is the right place to discuss it. Is that at all useful to point to, in communication that does not presume familiarity with GitHub discussions? 

_(If it's on a slide that says "we heard you, and we'll circle back around to this after the charter is approved", maybe that appropriately defers some discussions at the mike during the BOF?)_

**Review by ihlar — APPROVED** [review]
Looks good. 
The only wording I have a tiny concern about is the "fine-grained congestion signaling". 
Can this lead us down discussions on whether this is coarse grained congestion signaling or something else?

File: documents/charter.md [hunk 1]
```diff
@@ -41,6 +41,13 @@ The mechanism must be scalable and implementable by Internet infrastructure as i
 1. Security.
 The mechanism will have the ability to invoke security mechanisms that provide confidentiality, integrity, and authenticity of the communication. The working group will consider the value and implications of different confidentiality modes of the communication.
 
+The following topics are out of scope for SCONEPRO:
```
Comment (zaheduzzaman) [inline documents/charter.md:44]: I will perhaps rephrase it as below 

- Support for streaming video flows carried in other transports and substrates than QUIC.
- Support for other media types that would require awareness of additional network attributes beyond the attributes applicable to ABR video.
- Support for general purpose network attributes. If additional network attributes are identified, the working group will request recharter to add them to SCONEPRO.
- Support for congestion signaling from the network. SCONEPRO should not be treated as mechanism to replace congestion control or rate adaptation.

Comment (ihlar) [reply documents/charter.md:44]: I very much like and agree with that last bullet. It should dispel many potential misunderstandings.

Comment (SpencerDawkins) [reply documents/charter.md:44]: @zaheduzzaman - I've made this change, and don't object to merging the edited list, but I do wonder if streaming video flows carried in H3 transport is **clearly** in scope (or out of scope) for this work. If it's clearly in scope, that's cool, but if anyone else is uncertain, we should talk before the BOF!

**Pull request merged**